src/components/Todo.jsx

```diff
@@ -9,10 +12,13 @@ export function Todo() {
 
     return (todos.map((todo)=>(
           <div key={todo.id}>
+              <input type="checkbox" checked={todo.status}></input>
```

Comment:
make a separate common component for input field, considering there can be more than one input fields.

Reply:
can you please eleborate on this a bit.

Reply:
as we discussed in meeting